applications/NXmx.nxdl.xml

```diff
@@ -139,7 +139,7 @@
                         <dim index="1" value="nP" />
                         <dim index="2" value="i" />
                         <dim index="3" value="j" />
-                        <dim index="4" value="k" required="false"/>
```

Comment:
@RubelMozumder we have to reject these changes because it would change the behavior of an appdef from other people which is not our authority

Reply:
fixed, outdated information we checked the file directly and confirmed its state is correct